operator/pkg/central/values/translation/translation.go

```diff
@@ -137,14 +162,30 @@ func getCentralDBPersistenceValues(p *platform.DBPersistence) *translation.Value
 	return &persistence
 }
 
-func getCentralPersistenceValues(p *platform.Persistence) *translation.ValuesBuilder {
+func getCentralPersistenceValues(p *platform.Persistence, pvcExists func(claimName string) bool, pvcObsoleted bool) *translation.ValuesBuilder {
```

Comment:
I am not ready to checking API here because I am not ready to fetch the claimName here even though it may be available here. I need to claimName from pvc and obsolete, ctrlClient, and ctx from translate. Other choices are either passing all of them through args, or checking API prematurely.